ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>This is test 4.4</p><p> </p><p>For the steps in 4.3 add here annotations that will be send to the EFD and a not along the lines: "The constant speed test 4.3 will be executed using the TMA."</p><p> </p><p>How is the change of configuration parameters done eg for :<em>Parameters_upperPosLimit_deg</em> and <em>Parameters_RangeMaximum_deg</em></p><p><em>Can we add a description here on how the configuration parameters got changed and where, even if it is not part of the python code itself to not forget it?</em></p><p>Step done by the EUI should be included in the plots.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>I am not sure an easy way to do this programmatically to get the EUI data from EFD. You need to hard-code the time to subscribe the EFD data.</p>

Reply:
Updated in the commit: 137c4fd to f4b262b already. There is the case-sensitive error in your folder and I could not reach the LSP at summit and Tucson Teststand to rewrite the history. Stay like this at this time. Thanks!